app/src/views/features/sessions/SessionViewer/SessionPropertiesPanel.tsx

```diff
@@ -29,7 +29,7 @@ const SessionPropertiesPanel: React.FC<Props> = ({ getCurrentTimeOffset }) => {
   const isReadOnly = useSelector(getIsReadOnly);
   const sessionDescription = useSelector(getSessionRecordingDescription);
 
-  const recordingLengthInSeconds = useMemo(() => Math.floor(attributes.duration / 1000), [attributes]);
+  const recordingLengthInSeconds = useMemo(() => Math.floor(attributes?.duration ?? 0 / 1000), [attributes]);
```

Comment:
Why `0` as a fallback. In which cases `attributes` is undefined?

Reply:
It was breaking, when imported session on draft page is shown and user refreshes the page, now fixed with navigating to index page.